addons/project_todo/controllers/project_todo.py

```diff
@@ -0,0 +1,20 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import http
+from odoo.http import request
+
+
+class TodoController(http.Controller):
+
+    @http.route('/project_todo/new', type='json', auth='user')
```

Comment:
Equivalent to:
```suggestion
from odoo.http import Controller, request, route


class TodoController(Controller):

    @route('/project_todo/new', type='json', auth='user')
```

Not mandatory changes.